Example/SourcePointMetaApp/Controllers/AddPropertyViewController/AddPropertyViewController.swift

```diff
@@ -277,13 +277,23 @@ class AddPropertyViewController: BaseViewController, TargetingParamCellDelegate,
         for targetingParam in targetingParams {
             targetingParameters[targetingParam.targetingKey!] = targetingParam.targetingValue
         }
-        consentViewController = GDPRConsentViewController(accountId: Int(propertyDetails.accountId), propertyId: Int(propertyDetails.propertyId), propertyName: try! GDPRPropertyName(propertyDetails.propertyName!), PMId: propertyDetails.privacyManagerId!, campaignEnv: campaign, targetingParams: targetingParameters, consentDelegate: self)
-        if let messageLanguage = propertyDetails.messageLanguage {
-            consentViewController?.messageLanguage = addpropertyViewModel.getMessageLanguage(countryName: messageLanguage)
+        if let propertyName = propertyDetails.propertyName, let propertyId = propertyDetails.privacyManagerId {
```

Comment:
What do you think of extracting this validation logic into its own method, eg. `validatePropertyDetails`?

Reply:
Sure @andresilveirah will extract logic into `validatePropertyDetails` method